src/runtime/virtcontainers/kata_agent.go

```diff
@@ -1714,6 +1721,45 @@ func (k *kataAgent) createBlkStorageObject(c *Container, m Mount) (*grpc.Storage
 	return vol, err
 }
 
+// because all the info for secure mount passed from csi driver is store in mountinfo,
+// we extract mountinfo from json file to generate grpc request to mount external storage
+func (k *kataAgent) handleSecureMounts(c *Container, spec *specs.Spec) ([]*grpc.Storage, error) {
+    var volumeStorages []*grpc.Storage
+    for i, m := range c.mounts {
+        if m.Type == "secure_mount" {
```

Comment:
Hi @LindaYu17 I would like to know who（which component?） set `m.Type="secure_mount"`, which `Type` is for filesystem type.

Reply:
we add this type for secure mount external storage for CoCo, such as OSS.
We do mounting from CDH: https://github.com/confidential-containers/guest-components/pull/345
currently we only support alibaba-cloud-oss, and more storage could be supported.